### PR TITLE
Package range.0.8

### PR DESCRIPTION
**Pull request by aldrikfr**
### `range.0.8`
Fold on integer range
Library to work on range of sequential integers with features like : split, map,
filter capacities.
Designed for making distributed computing easier with libraries like functory.

---
* Homepage: https://github.com/aldrikfr/range
* Source repo: git+https://github.com/aldrikfr/range
* Bug tracker: https://github.com/aldrikfr/range/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by aldrikfr** [comment]
I'm not sure of the problem, travis isn't able to grab sources. I'm not sure of how I can fix that.

**Comment by mseri** [comment]
The url stanza is missing in the opam file

**Comment by mseri** [comment]
I am rerunning the test

**Comment by kit-ty-kate** [comment]
I'm noticing a couple of issues in your project. First there is no `dune-project` file, without this your package might break with the release of dune 2.0 or later. Also your opam file in the source repository contains a `url` section, this shouldn't be the case. `url` sections are only there for releases, not for dev packages. If someone wants to try your package using `opam pin add range --dev` for instance or simply `opam pin add .` they will get your archive and not the dev repo.

Could you retag your project with those fixes?

File: packages/range/range.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+maintainer : "Aldrik KLEBER <contact@aldrik.net>"
+authors : "Aldrik KLEBER"
+license : "GPL-3.0-only"
+homepage : "https://github.com/aldrikfr/range"
+dev-repo: "git+https://github.com/aldrikfr/range"
+bug-reports :"https://github.com/aldrikfr/range/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "base" { >="v0.12" & <"v0.14"}
+  "dune" {>= "1.8"}
```
Comment (kit-ty-kate) [inline packages/range/range.0.8/opam]: This should match the `dune-project` once it's there.

Comment (aldrikfr) [reply packages/range/range.0.8/opam]: > This should match the `dune-project` once it's there.

I'm not sure of what you mean

**Comment by aldrikfr** [comment]
> I'm noticing a couple of issues in your project. First there is no `dune-project` file, without this your package might break with the release of dune 2.0 or later. Also your opam file in the source repository contains a `url` section, this shouldn't be the case. `url` sections are only there for releases, not for dev packages. If someone wants to try your package using `opam pin add range --dev` for instance or simply `opam pin add .` they will get your archive and not the dev repo.
> 
> Could you retag your project with those fixes?

Yes ! Done ;)

File: packages/range/range.0.8/opam (hunk 1 above)
Comment (mseri) [reply packages/range/range.0.8/opam]: That the lower bound here reflects the version used in https://github.com/aldrikfr/range/blob/0.8/dune-project#L1

Comment (mseri) [reply packages/range/range.0.8/opam]: It's fine now

**Comment by mseri** [comment]
Thanks

**Pull request merged**